### PR TITLE
Customize for banner  EOL product versions

### PR DESCRIPTION
**Pull request by LucasSaintarbor**
Customize banner for EOL product versions.
- Banner message should mention product version is EOL and provide link to support matrix. 
- Resource: https://docusaurus.io/docs/next/swizzling#ejecting
- Resolves https://github.com/harvester/harvester/issues/3696

### CODE REVIEW

**Review by vickyhella — APPROVED** [review]
LGTM! Once this PR is merged, I will update the version banner for the zh locale as well. Thanks!

**Comment by LucasSaintarbor** [comment]
> LGTM! Once this PR is merged, I will update the version banner for the zh locale as well. Thanks!

@vickyhella thanks for the review and adding this for the zh locale!

**Pull request merged**
